Quadriga/src/main/java/edu/asu/spring/quadriga/qstore/impl/QStoreConnector.java

```diff
@@ -284,12 +413,9 @@ protected HttpHeaders buildRestHeader(List<HttpMessageConverter<?>> messageConve
         return headers;
     }
 
-
     private String getAuthHeader() {
-        String auth = env.getProperty("qstore.admin.username") + ":"
-                + env.getProperty("qstore.admin.password");
-        byte[] encodedAuth = Base64.encodeBase64(auth.getBytes(Charset
-                .forName("US-ASCII")));
+        String auth = env.getProperty("qstore.admin.username") + ":" + env.getProperty("qstore.admin.password");
+        byte[] encodedAuth = Base64.encodeBase64(auth.getBytes(Charset.forName("US-ASCII")));
```

Comment:
this is a bad idea, you can't necessarily make the assumption that this will be the correct charset. For now, I would just use the default platform charset, and we'll set only if this fails. (and then it should probably be utf-8)